### PR TITLE
 feat: ✨ added UI for publishing versions to datasets

### PR DESCRIPTION
**Pull request by Aydawka**

### CODE REVIEW

File: src/types/Version.d.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import type { StudyOverallOfficial } from "@/types/Study";
+import StudyConditions from "@/views/study/metadata/conditions/StudyConditions.vue";
```
Comment (megasanjay) [inline src/types/Version.d.ts:2]: What is this for?

File: src/views/study/dataset/publish/versions/PublishAllVersions.vue [hunk 2]
```diff
@@ -4,7 +4,7 @@ import type { Version } from "@/types/Version";
 import { displayHumanFriendlyDateAndTime } from "@/utils/date";
 
 // import { baseURL } from "@/utils/constants";
-const baseURL = "http://localhost:3001/api";
+const baseURL = "http://localhost:5000";
```
Comment (megasanjay) [inline src/views/study/dataset/publish/versions/PublishAllVersions.vue:7]: import from the utils

**Review by megasanjay — CHANGES_REQUESTED** [review]

File: src/components/header/AppHeader.vue [hunk 3]
```diff
@@ -88,15 +90,6 @@ const handleSelect = (key: string | number) => {
             </n-button>
           </RouterLink>
 
-          <RouterLink to="/auth/signup" v-if="!authStore.isAuthenticated">
```
Comment (megasanjay) [inline src/components/header/AppHeader.vue:91]: Why was this button removed?

File: src/assets/css/main.css [hunk 4]
```diff
@@ -155,3 +155,14 @@ Transition styles for <transition />
...
+li.related-item:nth-child(even){
+  background: rgb(250, 250, 252)
+}
+
+td.related-item{padding:0;}
+
+li.related-item{
+  padding: 12px;
+  border-left: rgb(239, 239, 245)
+}
```
Comment (megasanjay) [inline src/assets/css/main.css:168]: These should be added in tailwind directly on the component. You can dynamically assign classes via `:class=""`

Comment (Aydawka) [reply src/assets/css/main.css:168]: can tailwind gray replace this specific strip tone of rgb(250, 250, 252)?

File: src/components/header/AppHeader.vue (hunk 3 above)
Comment (Aydawka) [reply src/components/header/AppHeader.vue:91]: i don't think i removed, either accidentally, or i merged my branch with staging before push, it came from there

File: src/assets/css/main.css (hunk 4 above)
Comment (megasanjay) [reply src/assets/css/main.css:168]: I don't think you need the exact value. The closest tailwind color should match. But if you wanted to: https://tailwindcss.com/docs/border-color#arbitrary-values

**Review by megasanjay — APPROVED** [review]

**Pull request merged**